trade-finance-manager-ui/server/controllers/utilisation-reports/edit-payment/index.ts

```diff
@@ -24,6 +24,8 @@ type GetEditPaymentRequest = CustomExpressRequest<{
   };
 }>;
 
+const template = 'utilisation-reports/edit-payment.njk';
```

Comment:
Please can we use a more descriptive name. We also refer to the problem with service template so there are multiple templates going around, will also make the render line more readable.

```suggestion
const editPaymentPageTemplate = 'utilisation-reports/edit-payment.njk';
```

Reply:
Same for other similar usages of the word 'template'

Reply:
I think a specific name in controllers would make things harder to maintain moving forward. E.g if we call this `editPaymentPageTemplate` and the page renames to "modify payment" we'd have to rename all such variables, or they could be missed

I think `template` is succinct enough as the parent directory is called `edit-template`. There should never be more than one template referenced in a controller other than "problem with service" which is self explanatory. Maybe in another PR all template references could be moved into constants and imported so e.g

```
TEMPLATES.EDIT_PAYMENT
TEMPLATES.PROBLEM_WITH_SERVICE
```

Reply:
It would be good to have all the templates in a constants file but that will be for later. 
Where there are multiple templates in a file I have made sure the const is more descriptive